Meetings/Meeting.ics

```diff
@@ -1,67 +1,150 @@
 BEGIN:VCALENDAR
 PRODID:-//Microsoft Corporation//Outlook 16.0 MIMEDIR//EN
 VERSION:2.0
-METHOD:PUBLISH
+METHOD:REQUEST
 X-MS-OLK-FORCEINSPECTOROPEN:TRUE
 BEGIN:VTIMEZONE
-TZID:UTC
+TZID:Central Standard Time
```

Comment:
Does changing the TZ from UTC to CST get us (rather "the rest of us") in trouble with DST (i.e. when a switch to CDT occurs) ?